### ISSUE TITLE
WCAG 1.2.3 is missing in report

### ISSUE THREAD

**Issue by Bram-diederik** [microsoft/accessibility-insights-web#3414]
I did a full report for a website and found out one WCAG item (WCAG 1.2.3) is missing in the report. 
The application states that is does a full AA test of a website but if this is missing this is not the case. 

please find the attached generated in the zip and see that WCAG 1.2.3 is missing  

[PuberGezond_Assessment_20200915_Home_nr1.zip](https://github.com/microsoft/accessibility-insights-web/files/5297980/PuberGezond_Assessment_20200915_Home_nr1.zip)

**Comment by ferBonnin** [comment]
Hi @Bram-diederik thanks for the bug report!

In Assessment, the test _Multimedia_ > requirement _Audio description_ maps to both 1.2.3 and 1.2.5. If you meet 1.2.5, you also meet 1.2.3.  Here’s a clarification from [Understanding Success Criterion 1.2.5: Audio Description (Prerecorded)](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded.html):

1.2.3, 1.2.5, and 1.2.8 overlap somewhat with each other. This is to give the author some choice at the minimum conformance level, and to provide additional requirements at higher levels. 
* At Level A in Success Criterion 1.2.3, authors do have the choice of providing either an audio description or a full text alternative. 
* If they wish to conform at Level AA, under Success Criterion 1.2.5 authors must provide an audio description - a requirement already met if they chose that alternative for 1.2.3, otherwise an additional requirement. 


Leaving this bug open to ensure the requirement mentioned above also has WCAG 1.2.3 in the report.

**Comment by ferBonnin** [comment]
Hi @Bram-diederik this is now fixed! Please take a look at the fix in the [Canary version of Accessibility Insights for Web](https://chrome.google.com/webstore/detail/accessibility-insights-fo/hbcplehnakffdldhldncjlnbpfgogbem) and let us know what you think!
